main_routers/workshop_router.py

```
@@ -21,17 +21,123 @@
 router = APIRouter(prefix="/api/steam/workshop", tags=["workshop"])
 # 全局互斥锁，用于序列化创意工坊发布操作，防止并发回调混乱
 publish_lock = threading.Lock()
 logger = logging.getLogger("Main")
 
+
+def get_workshop_meta_path(character_card_name: str) -> str:
+    """
+    获取角色卡的 .workshop_meta.json 文件路径
+    
+    Args:
+        character_card_name: 角色卡名称（不含 .chara.json 后缀）
```

Ruff (0.14.10)

Docstring contains ambiguous `（` (FULLWIDTH LEFT PARENTHESIS). Did you mean `(` (LEFT PARENTHESIS)?

(RUF002)